### PR TITLE
Show a diff of the PKGBUILD when updating a package from AUR

### PR DESCRIPTION
**Pull request by larchunix**
For devel packages compare the last two revisions of the PKGBUILD as a fallback.

Implements #83.

Feedback welcome.

### CODE REVIEW

**Comment by Skunnyk** [comment]
Nice works ! 

Some notes: 
It add 2 extra "line return" between the package name and the " ( Unsupported package: Potentially dangerous ! )" warning.
screenshot_2016-01-01_18-06-05


And maybe enable/disable this feature with a new rc option ?

**Comment by larchunix** [comment]
@Skunnyk 
> It add 2 extra "line return" between the package name and the " ( Unsupported package: Potentially dangerous ! )" warning.

I can't reproduce (it should happen only when the diff is empty which is not the case for `sky`). I updated my PR several times,  did you test with the latest version ?

> And maybe enable/disable this feature with a new rc option ?

PR updated. New option added: `AURSHOWDIFF`.

File: src/lib/aur.sh [hunk 1]
```diff
@@ -23,8 +23,9 @@ aur_get_pkgbuild() {
 
 	if ((local_aurusegit)); then
 		local git_repo_url=$(pkgquery -Aif "%G" "$pkg")
+		((AURSHOWDIFF)) || local _arg="--depth=1"
 		# We're already in "$pkg"/ here, so clone to the current directory
-		git clone "$git_repo_url" . || return 1
+		git clone $_arg "$git_repo_url" . || return 1
```
Comment (Skunnyk) [inline src/lib/aur.sh]: Woops, the shallow clone thing break the git clone :)
```
++ package-query --aur-url https://aur.archlinux.org -Aif %G sky
+ local git_repo_url=https://aur.archlinux.org/sky.git
+ (( AURSHOWDIFF ))
+ git clone sky https://aur.archlinux.org/sky.git .
Too many arguments.
```

Comment (larchunix) [inline src/lib/aur.sh]: Shame on me. Fixed.

**Comment by Skunnyk** [comment]
Sound good, I have done some tests and it works :)

**Comment by Skunnyk** [comment]
Let's go \o/
Tip: You must use it with AURUSEGIT=1 and AURSHOWDIFF=1 in yaourtrc

**Pull request merged**